src/components/about/content-geniza-en.jsx

```diff
@@ -0,0 +1,15 @@
+      <p className="text">A geniza is a temporary storeroom or repository for old, used, and/or damaged Torah scrolls and other texts regarded as sacred in the Jewish tradition. The practice of not throwing away such documents is based on the sanctity imputed to them because they contain a written form of God’s name in Hebrew characters. Jewish custom dictates that these sacred writings be treated respectfully and buried when no longer in use. For reasons not entirely clear, Jews affiliated with the Ben Ezra synagogue kept their used texts in their Geniza chamber for nearly a millennium rather than burying them. There are similar practices of storing documents among Muslims and Christians across the Islamic world, as exemplified for example in the <a href="http://www.academia.edu/6120191/Geniza_for_Islamicists_Islamic_Geniza_and_the_New_Cairo_Geniza" target="_blank">Sana'a manuscripts</a> rediscovered in the attic of the Great Mosque of Sana'a during a 1972 renovation, but the Cairo Geniza is the largest cache of such stored documents.</p>
+      <p className="text">The Cairo Geniza corpus includes both traditional religious, philosophical, and scientific texts (often termed the “literary” Geniza), and a wide range of everyday documents, such as letters, tax rolls, contracts, legal testimonies and other court documents, receipts, accounts and lists, known collectively as the “documentary” geniza. The survival of documents in the Cairo Geniza is of exceptional importance today because of what these fragmentary remains can teach us about the everyday lives of medieval Jews, Christians and Muslims and the relationships among them. The literary and documentary fragments together also offer unexpectedly fine-grained information about the historical context in which most of them were written and circulated: Cairo during the Fatimid and Ayyubid periods, one of the most important cultural and economic centers of its day.</p>
+      <p className="text">Beginning in the Renaissance, European diplomats, scholars, and travelers to the Middle East and North Africa generated a vibrant antiquities market as they actively sought and purchased texts and artifacts to bring home with them for study or to add to their personal libraries. During the 19th century, the contents of the Cairo Geniza began to draw attention when assorted fragments began to come up for sale. Orientalists, traders, collectors, and scholars amassed these fragments for their own collections. Today, Cairo Geniza fragments can be found in collections from throughout the world. To learn about the history of the collections the fragments you are transcribing come from, <a href="https://docs.google.com/document/d/1uxY_EgX9qUp-oQMFbFboZvMs3t-nd9FLLibhVO37xdA/edit?usp=sharing" target="_blank">click here</a> or scroll down.</p>
```

Comment:
This link isn't in the Arabic translations. Also, should this be a Google doc?

Reply:
I have _several questions_ about the translations, and yes, this is one of them.
- Check out [the translations document](https://docs.google.com/document/d/1YckwDXbWd_NlvwHLGo4AxHcWsIBw3ZHHlTRqbpLhEUA/edit#) (Google Docs, Zooniverse login required)
- The English version links to Google Docs, and actually describes details similar to the Provenance section.
- The Hebrew version helpfully marks where the last link should appear... but only has 3 paragraphs to contrast the 5 paragraphs of English
- The Arabic version has no markings to indicate where links should appear.

The differences are pretty stark once you compare the `content-geniza-LANG.js` files side by side. 🤷‍♂️

Reply:
I think we should remove this as it seems awkward to link to a Google document on our page. @snblickhan - do you agree with this? The about section contains the following sentence with link in English and Hebrew "To learn about the history of the collections the fragments you are transcribing come from, [click here](https://docs.google.com/document/d/1uxY_EgX9qUp-oQMFbFboZvMs3t-nd9FLLibhVO37xdA/edit) or scroll down."

Reply:
We don't want to link to any Google docs anywhere.

The "click here" text was included in the event that we had links to sub-sections within the About document -- it should link to the Provenance section of the About page.